### DISCUSSION TITLE
MAME 0.272

### DISCUSSION THREAD

**Discussion by cuavas** [mamedev/discussions#165] [Announcements]
# [MAME 0.272](https://www.mamedev.org/?p=544)

The end of the year is almost upon us, but before that happens, make sure you try MAME 0.272!  We’re very happy to announce that the early SNK game Tangram Q is now supported.  This game was never widespread, and working examples are rarer than ever.  MAME now gives you the unprecedented opportunity to experience this piece of gaming history.  Another obscure system that’s starting to take shape is the Estonian EKTA Juku.  This 8-bit educational computer was powered by a Soviet 8080 clone and ran the CP/M-derived EKDOS operating system.

The French Thomson computers have been receiving some attention lately, and software compatibility is starting to improve.  Progress on the NEC PC-98 family is still coming along nicely.  Our CD-ROM drive emulation has been getting better, which has a positive impact on numerous computers as well as game systems.  The Epoch Super Cassette vision also got some nice fixes this month.

Numerous reported issues were fixed in this release, including some long-standing issues that had eluded developers for years.  There are lots of new working bootleg arcade games to try out.  Several TV games have been promoted to working this month, too.

You can read about all the exciting work that went into this release in the [whatsnew.txt file](https://www.mamedev.org/releases/whatsnew_0272.txt).  As always, the source code and 64-bit Windows binary packages are available on [the download page](https://www.mamedev.org/release.html).

[Read the rest of this entry »](https://www.mamedev.org/?p=544#readmore)

**Comment by Gently28** [comment]
Dear cuavas,this week  I was playing some of Dynax's mahjong games and I found that there are still a lot of DIP-SW Unknown in MAME0.272,,and i can't fixed it, so i hope you can  do if you free,thank you. please check and adds at next ver .
https://1drv.ms/x/c/9f5d48a4358b200d/EWeLj0XEToRPrh6HqHf9-bUBtQYGcjjXqQI-IhcePJNppg?e=Byaouh

**Reply by cuavas** [reply]
I’ve updated the DIP switches for the games where there are images of the relevant pages from the manuals.  I’m not going to change the ones where you have not provided legible images of the manuals as there are numerous transcription errors and incorrect descriptions of settings in the spreadsheet.

In the future, it would be far more helpful if you could just post the best quality images you have of the manuals rather than insisting on burying them inside spreadsheets.

**Comment by cuavas** [comment]
Bob Zed’s video highlighting some of the work in this release:
[![MAME 272 - What's new](https://img.youtube.com/vi/ar0lDjS7e8M/0.jpg)](https://www.youtube.com/watch?v=ar0lDjS7e8M)

**Comment by Gently28** [comment]
https://mametesters.org/view.php?id=8995
tenkai
the black is"Show In-Game Clock",it is mean”Plug it in，time is began  in-game ",and you see a clock . it is SW3-10 Key ON.

**Comment by Gently28** [comment]
I have a question:the key "Flip_Screen"  is void in Dynax mahjong"  Mahjong
The Mysterious Orient ”麻雀世界の神秘  (mjmyster)，include 麻雀東洋の神秘、麻雀世界の神秘等系列，i
look at the key is:
PORT_DIPNAME( 0x80, 0x00, DEF_STR(Flip_Screen) )
 PORT_DIPLOCATION("SW 1:10")
PORT_DIPSETTING(    0x00, DEF_STR(Off) )
PORT_DIPSETTING(    0x80, DEF_STR(On) ),
i guess it maybe is
PORT_DIPNAME( 0x80, 0x80, DEF_STR(Flip_Screen) )
 PORT_DIPLOCATION("SW 1:10")
PORT_DIPSETTING(    0x00, DEF_STR(Off) )
PORT_DIPSETTING(    0x80, DEF_STR(On) ),
is it no or right? please check and fixed.
and i already 整理完神秘系列的式样书，i will sending you at later,ok?  i hopr you well
fixed they,ok?😋 sorry,excuse me.

Vas Crabb ***@***.***> 于2024年12月15日周日 01:31写道：

**Reply by cuavas** [reply]
Screen flipping is not emulated for these games.  Changing the DIP switch labels won’t make any difference.  Screen flipping emulation needs to be added.
